app/templates/vendor/orders.html

```diff
@@ -105,7 +108,22 @@ <h2 class="ui header">
         {% endif %}
 
     </div>
+    <div class="ui modal">
+        <div class="header">You are about to <b class="action-type"></b> this order:</div>
+        <div class="content">
+            <div class="ui form">
+                  <div class="field">
+                    <label>Please write any comments to be sent to the customer <em>(optional)</em>:</label>
+                    <textarea rows="2" class="comment-container"></textarea>
+                  </div>
+            </div>
+        </div>
 
+        <div class="confirm-action">
+            <button class="negative ui button tiny compact cancel">Cancel</button>
```

Comment:
Just a UI thing - but I'm wondering about the ordering of the Cancel and Approve buttons? (the same for the Merchant side when they place an order)

I feel like Approve should be "highlighted" since the buttons we have initially that takes them to the dialog seem clear and that vendor's would be intentionally clicking one (and the color also helps). Would love to hear thoughts!